src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public static final int MY_CONSTANT = 100;
```

Comment:
Name of constant or variable should give us understanding of what is it for.
NUMBER_LIMIT, MAX_BALL_NUMBER, MAX_NUMBER or anything else that makes sense

Reply:
Done.